rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -709,7 +713,7 @@ def _maybe_decode_erc20_721_transfer(
                 action_items.pop(idx)
                 break  # found an action item and acted on it
 
-        # Add additional information to transfers for different protocols
+        # Add additional information to transfers for different counterparties
```

Comment:
As I mentioned here: https://github.com/rotki/rotki/pull/5726/files#r1138545263

Use the word protocols, irrespective of the variable name
```suggestion
        # Add additional information to transfers for different protocols
```